### ISSUE TITLE
Convert ckpt file to nemo

### ISSUE THREAD

**Issue by SagyHarpazGong** [NVIDIA/NeMo#7131]
Hi.

I trained msdd model but I stopped the training (not waited until the max_epoch value), is there any script to convert the ckpt file into nemo file, the ckpt file contain all the optimizer params and gradients.

please advise.

**Comment by gabitza-tech** [comment]
Hi,

**Comment by gabitza-tech** [comment]
Hi,
    
You can initialize the model from a pretrained checkpoint and then save it to **.nemo** format:

```
import torch
torch.cuda.empty_cache()
import pytorch_lightning as pl
from omegaconf import OmegaConf
from pytorch_lightning import seed_everything

from nemo.collections.asr.models import EncDecDiarLabelModel
from nemo.core.config import hydra_runner
from nemo.utils import logging
from nemo.utils.exp_manager import exp_manager
import copy
seed_everything(42)

@hydra_runner(config_path="../conf/neural_diarizer", config_name="msdd_6scl_30_05_50Povl_256x3x32x2.yaml")
def main(cfg):
    logging.info(f'Hydra config: {OmegaConf.to_yaml(cfg)}')
    trainer = pl.Trainer(**cfg.trainer)
    exp_manager(trainer, cfg.get("exp_manager", None))

    msdd_model = EncDecDiarLabelModel(cfg=cfg.model, trainer=trainer)
    #logging.info('WE LOADED THE CONFIGURATION')
    msdd_model.maybe_init_from_pretrained_checkpoint("nemo_models/msdd_full_dev.ckpt")
    save_path = f"msdd.nemo"
    msdd_model.save_to(f"{save_path}")
```
Let me know if it worked for you!
